api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -100,6 +104,67 @@ void mergeResearchPurposeIntoWorkspace(
   WorkspaceResource workspaceResourceFromDbWorkspace(
       DbWorkspace dbWorkspace, WorkspaceAccessLevel accessLevel);
 
+  @Mapping(target = "workspaceFirecloudName", source = "dbWorkspace.firecloudName")
```

Comment:
@jaycarlton do you know if there's a way to avoid all this repeated boilerplate? It feels like it should be abstractable.

Reply:
When the corresponding fields are named the same in the source and target, it just works.